File: 2A_1/python/funkcje02.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  funkcje02.py
#  

licz1 = int(input("Podaj liczbę: "))  # zmienna globalna
licz2 = int(input("Podaj liczbę: "))  # zmienna globalna

def sumuj(l1, l2):
    """
    Funkcja sumuje dwie podane liczby i drukuje wynik.
    """
    suma = l1 + l2  # zmienne lokalne
    print("Suma:", suma)


def sumuj2():
    print("Suma:", licz1 + licz2)


def main(args):
    # global licz1, licz2
    licz1 = 0
    licz2 = 3
    
    print("Suma:", licz1 + licz2)
    sumuj2()  # wywołanie funkcji
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2A_1/cpp/funkcje02.cpp
/*
 * funkcje02.cpp
 * 
 */


#include <iostream>

using namespace std;

int liczba1 = 0;
int liczba2 = 0;  // zmienne globalne

void sumuj() {
	int suma = liczba1 + liczba2;
	cout << "Suma: " << suma << endl;
}

int main(int argc, char **argv)
{
	// int liczba1, liczba2;  // zmienne lokalne
	cout << "Podaj liczby: ";
	cin >> liczba1;
	cin >> liczba2;
	cout << "Suma: " << liczba1 + liczba2 << endl;
	
	sumuj();
	
	return 0;
}

